### PR TITLE
examples: small fixes

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

This fixes a few small errors in the documentation and adds the option to use 1024 byte blocks in `examples/suit_update`

### Testing procedure

`examples/suit_update` still works with `LARGE_BLOCKS=1`

### CODE REVIEW

File: examples/suit_update/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Table of contents:
 
 - Install python dependencies (only Python3.6 and later is supported):
 
-      $ pip3 install --user cbor2 cryptography
+      $ pip3 install --user cbor2 cryptography LinkHeader
```
Comment (chrysn) [inline examples/suit_update/README.md:36]: Any reason not to just install `aiocoap[linkheader] >= 0.4.1`?

Comment (benpicco) [reply examples/suit_update/README.md:36]: I didn't know this syntax, what does it do?

**Comment by chrysn** [comment]
It selects "extras" (optional dependencies) of the package. Concretely, it's a way of telling the package "I want to use link-format features", and if a newer version implements that using other means than the LinkHeader package, using the extra makes that work. (If, however, and I didn't check that yet, Riot code uses the LinkHeader module directly, what's in the PR is fine).

https://aiocoap.readthedocs.io/en/latest/installation.html describes aiocoap's extras.

**Comment by chrysn** [comment]
Why is CONFIG_GNRC_PKTBUF_SIZE only set in the non-large-blocks case? Seems to me the default would be good for both (although the small case could work with a smaller one)

**Comment by benpicco** [comment]
I'm not sure why `CONFIG_GNRC_PKTBUF_SIZE` was reduced here from the default 6144 Bytes.
But with 1024 byte CoAP blocks instead of 128 Bytes, this is no longer enough.

**Comment by chrysn** [comment]
On Sat, Apr 10, 2021 at 12:29:37PM -0700, benpicco wrote:
> I'm not sure why `CONFIG_GNRC_PKTBUF_SIZE` was reduced here from the default 6144 Bytes.
> But with 1024 byte CoAP blocks instead of 128 Bytes, this is no longer enough. 

Sorry, I didn't find that default earlier when I did the reviewing from
mobile -- kind of makes sense. I'd still like to understand better why
the 2kiB buffer is insufficient (there should only ever be one message
in flight, one of them being the requested 1200 byte data chunk, and the
request that's still around while that is being received into the 2k
receive buffer sits in the Gcoap retransmit buffer and not in GNRC).

(Good enough for the PR anyway; once the Python aspect is answered I can
ACK this.)

File: examples/suit_update/README.md (hunk 1 above)
Comment (benpicco) [reply examples/suit_update/README.md]: Ok, changed it.

**Review by chrysn — APPROVED** [review]
LGTM. Thanks, merging.

**Pull request merged**